CarpetLib/src/restrict_3d_dgfe_rf2.cc

```diff
@@ -118,8 +118,8 @@ void restrict_3d_dgfe_rf2(
   // Ensure we traverse an integer number of elements
   assert(all(regext % (ORDER + 1) == 0));
 
-// Loop over coarse region
-#pragma omp parallel for collapse(3)
+  // Loop over coarse region
+  // #pragma omp parallel for collapse(3)
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.